assets/xml/overlays/ovl_En_Clear_Tag.xml

```diff
@@ -0,0 +1,14 @@
+<Root>
+    <File Name="ovl_En_Clear_Tag" BaseAddress="0x809D35B0" RangeStart="0x26F0" RangeEnd="0x89F0" Segment="128">
+        <DList Name="gArwingDL" Offset="0x26F0"/>
+        <DList Name="gLaserDL" Offset="0x6388"/>
+        <DList Name="gArwingBackfireDL" Offset="0x6688"/>
+        <DList Name="gDebrisEffectMaterialDL" Offset="0x6A38"/>
+        <DList Name="gDebrisEffectDL" Offset="0x6AB8"/>
+        <DList Name="gFireEffectMaterialDL" Offset="0x7100"/>
+        <DList Name="gFireEffectDL" Offset="0x71A8"/>
```

Comment:
Those names seem a bit too generic and they will probably conflict with other things in the game, so I suggest using something more specific to Arwing (probably fine to just include Arwing in all these names tbh)

Reply:
Works for me.